pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -124,14 +122,16 @@ def _save_ibmq_auth(qiskit_config: Optional[QiskitConfig]) -> None:
     token = None
     if qiskit_config is not None:
         token = qiskit_config.ibmq_api_token
-    if not IBMQ.active_account():
-        if IBMQ.stored_account():
-            IBMQ.load_account()
+    if token is None and os.getenv("PYTKET_REMOTE_QISKIT_TOKEN") is not None:
```

Comment:
I am not sure if it is the best way to keep this in here, but I have not found a way without this. This is run locally for all users, maybe we should add a check for an additional env variable to make sure that this is only run on the CI?